datafusion/src/physical_plan/functions.rs

```diff
@@ -2575,6 +2575,19 @@ mod tests {
             Int32,
             Int32Array
         );
+        // #[cfg(feature = "regex_expressions")]
```

Comment:
Did you intend to fix this case?

Reply:
Ah, this one doesn't work and I have removed.

There is a bug in the regex handling https://github.com/apache/arrow-datafusion/issues/1429
I might pick up that one next.